### PR TITLE
completed task 5

### PR DESCRIPTION
**Pull request by a-stenhouse**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -26,4 +26,41 @@ describe("get /api/categories", () => {
...
+})
+
+describe("get /api/reviews/review_id", () => {
+    test("responds with JSON containing the expected review according to the review ID", () => {
+        return request(app)
+            .get("/api/reviews/1")
+            .expect(200)
+            .then((response) => {
+                expect(response.body.review).toMatchObject({
+                    review_id: expect.any(Number),
```
Comment (JVal-NC) [inline __tests__/app.test.js]: Would expect a relevant review_id number

File: app.js [hunk 2]
```diff
@@ -1,12 +1,24 @@
 const express = require('express');
 const app = express();
-const { getCategories } = require("./controller.js")
+const { getCategories, getReviewID } = require("./controller.js")
 
+app.use(express.json());
```
Comment (JVal-NC) [inline app.js]: Not needed yet, please remove.

File: app.js [hunk 3]
```diff
@@ -1,12 +1,24 @@
...
 
+app.get("/api/reviews/:review_id", getReviewID)
+
+
 app.use((err, req, res, next) => {
-    res.status(500).send("Server Error!")
+    if (err.status && err.msg) {
+        res.status(404).send({ msg: err.msg })
+    } else if (err.code === "22P02") {
+        res.status(400).send({ msg: "Not a valid user ID" })
```
Comment (JVal-NC) [inline app.js]: May cause issues later on, fine for now but just a heads up

**Review by JVal-NC — COMMENTED** [review]
Looks good, few minor changes to consider

File: app.js (hunk 3 above)
Comment (a-stenhouse) [reply app.js]: Okay, I'll refactor this later on

File: __tests__/app.test.js [hunk 4]
```diff
@@ -26,4 +26,41 @@ describe("get /api/categories", () => {
...
+})
+
+describe("get /api/reviews/review_id", () => {
+    test("responds with JSON containing the expected review according to the review ID", () => {
+        return request(app)
+            .get("/api/reviews/1")
+            .expect(200)
+            .then((response) => {
+                expect(response.body.review).toMatchObject({
+                    review_id: 1,
```
Comment (Augs0) [inline __tests__/app.test.js:38]: If we are trying to get a single review, we would hope the data would be exactly that data, so here we should check the exact title, etc.

File: app.js (hunk 3 above)
Comment (Augs0) [reply app.js]: ^ to clarify, this is because this code could happen in any table, not just user ids

**Review by Augs0 — CHANGES_REQUESTED** [review]
Just a little change to your test and then you're good to go :)

**Review by Augs0 — APPROVED** [review]
Good to merge, Arran! :D

**Pull request merged**